__tests__/app.test.js

```diff
@@ -3,3 +3,30 @@ const seed = require("../db/seeds/seed.js");
 const testData = require("../db/data/test-data/index.js");
 const request = require("supertest");
 const app = require("../app.js");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  test("status: 200 - responds with with an array of topics, each with a slug and description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.topics).toHaveLength(3);
+        body.topics.forEach((topic) => {
+          expect(typeof topic.slug).toBe("string");
+          expect(typeof topic.description).toBe("string");
+        });
+      });
+  });
```

Comment:
The test below should be in it's own describe block as it is testing ALL bad URLs and therefore isn't specific to the /api/topics endpoint. Note: there is no need to test for all bad URLs again